airsonic-main/src/main/java/org/airsonic/player/dao/UserDao.java

```diff
@@ -147,7 +151,7 @@ public void createUser(User user) {
      * @param username The username.
      */
     public void deleteUser(String username) {
-        if (User.USERNAME_ADMIN.equals(username)) {
+        if (settingsService.getAdminUser().equals(username)) {
             throw new IllegalArgumentException("Can't delete admin user.");
```

Comment:
Maybe show the username of the current adminUser?

Reply:
I'd be happy to add that if you think it's needed.  The only reason I didn't change the exception message is that I tend to prefer trying to keep minimal changesets (like avoiding unnecessary or unrelated formatting changes).

Reply:
I do agree with your reasoning, as I tend to do the very same, but in this case, the change is so trivial that I would prefer to see it done here, instead of having to review a second commit simply changing this line.

But on the other hand, I guess they'll be more commits on top of this to polish the feature a bit more, so feel free to ignore this comment :)

Reply:
In reality, with the changes from #1129, its very unlikely that this will ever happen, but I went ahead and added the username as requested.